docs/introduction/get-started/create-account-use-faucet.md

```diff
@@ -134,14 +134,14 @@ State synced to block 179672
 3. Find your account and note id by listing both `accounts` and `input-notes`:
 
       ```shell
-      miden-client account -l
-      miden-client input-notes -l
+      miden account -l
+      miden input-notes -l
       ```
 
 4. Consume the note and add the funds from its vault to our account using the following command: 
 
       ```shell
-      miden-client tx new consume-notes <Account-Id> <Note-Id>
+      miden tx new consume-notes <Account-Id> <Note-Id>
```

Comment:
Do I need to add the flags `--account` here?

as we describe in here?

https://github.com/0xPolygonMiden/miden-client/blob/next/docs/cli-reference.md#types-of-transaction

I think I confused Martins PR last time and some changes got merged to `main` and then overridden.

Reply:
Good point, hadn't taken those changes into account